bin/kube_overrides.rb

```diff
@@ -37,6 +43,28 @@
         env['valueFrom']['secretKeyRef']['name'] = current_secrets_name
       end
     end
+    overrides.each do |k, v|
+      child = container
+      k[0...-1].each do |elem|
+        child[elem] ||= {}
+        child = child[elem]
+      end
+      case child
+      when Array
+        # Deal with the environment list specially
```

Comment:
Why does this mention "the environment list specially" when it then applies the filtering to all arrays?

Reply:
Fixed in 7b3c931f0c7ec9d0f15e7fa29968b6b2a1baaa98; changed the condition to actually only work with the environment list.